_examples/pubsubs/aws-sns/main.go

```diff
@@ -0,0 +1,133 @@
+// Sources for https://watermill.io/docs/getting-started/
```

Comment:
I guess you can add i to to getting-started as well?

Reply:
bd3b9b253a88f4d57b09c6d69ef22fddd0eeb7cb